packages/wallets/alpha/src/adapter.ts

```diff
@@ -109,7 +107,7 @@ export class AlphaWalletAdapter extends BaseMessageSignerWalletAdapter {
 
             if (!wallet.isConnected) {
                 try {
-                    await wallet.connect();
+                    return await wallet.connect();
```

Comment:
I fixed this in master, but this should not return here. It breaks the `connect` function of the adapter, which is expected to dispatch events and do setup at the end.